akka-actor/src/main/resources/reference.conf

```diff
@@ -420,6 +420,7 @@ akka {
         mailbox-type = "akka.dispatch.UnboundedMailbox"
       }
 
+      # deprecated (because relies on blocking)
```

Comment:
I can't see any. `capacity` and `pushTimeOut` always comes together. I think the ticket is about emitting a warning when pushTimeout != 0. Nothing should be deprecated as far as I can see. Well, we could start removing the `pushTimeOut` parameter and (use 0 as default). That would be very complicated to do in a binary compatible way, since the mailboxes are case classes :-(